components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -6554,12 +6554,13 @@ public static JSONArray getAppsByHitCount(String userId,
 	 * @param userRoles          : user roles
 	 * @param isAnonymousAllowed : is anonymous access allowed to URL pattern
 	 * @param objPartialMappings : Object which contains XACML policy partial details arrays
+	 * @param policyGroupDesc    :Policy group Desciption
 	 * @return : last saved policy group id
 	 * @throws AppManagementException if any an error found while saving data to DB
 	 */
 	public static Integer savePolicyGroup(String policyGroupName, String throttlingTier,
```

Comment:
Added a jira https://wso2.org/jira/browse/APPM-1072
Similar issue will be fixed through out the code in future releases.